apps/server/src/modules/tldraw/controller/api-test/tldraw.controller.api.spec.ts

```diff
@@ -14,13 +16,23 @@ describe('tldraw controller (api)', () => {
 	let app: INestApplication;
 	let em: EntityManager;
 	let testApiClient: TestApiClient;
+	const API_KEY = '7ccd4e11-c6f6-48b0-81eb-cccf7922e7a4';
 
 	beforeAll(async () => {
 		const module: TestingModule = await Test.createTestingModule({
 			imports: [ServerTestModule],
 			controllers: [TldrawController],
 			providers: [Logger, TldrawService, TldrawRepo],
-		}).compile();
+		})
+			.overrideGuard(AuthGuard('api-key'))
+			.useValue({
```

Comment:
This is the legacy way to test. Please check class TestXApiKeyClient and places that us it.